CriticalMass/TimeTraveler.swift

```diff
@@ -10,27 +10,35 @@ class TimeTraveler {
         self.date = date
     }
 
-    func travelTime(by interval: TravelInterval, _ calendar: Calendar = .current) {
+    @discardableResult
+    func travelTime(by interval: TravelInterval, _ calendar: Calendar = .current) -> Self {
```

Comment:
This is odd. Looks like s builder pattern. But I can see that the API could be improved

Reply:
TimeTraveler was already kind of a Builder class before. This changes just adds the ability to chain the operations which makes it possible to use it in a single line to support property or argument default assignments. It's actually not necessary anymore because I removed the part that's relying on it in the meantime